src/components/Header.vue

```diff
@@ -35,18 +34,17 @@
 import { eventBus } from '../main.js';
 
 export default {
-    //props: ['shareLink'],
     data() {
         return {
-            shareLink: ''   // 0 = hiden
+            shareLink: ''   // 0 = textbox hiden
         }
     },
     methods: {
         nodeSubmit() {
             this.shareLink = "generating link...";
             eventBus.$emit('node-submit');
         },
-        shareCheck() {
+        shareCheck() {      // checking if share panel should be visible
             if (this.shareLink === 'hide') {
                 return false;
             } else {
```

Comment:
redudnant else, if you are using `if` + `return` you breaking everything else in if and outside if :)... also you could do some shorthand: 
```
shareCheck(){
   return this.shareLink !== 'hide'
}
```